src/components/Wishlist/wishContext.js

```diff
@@ -0,0 +1,43 @@
+import { createContext, useContext, useState, useEffect } from "react";
+const axios = require('axios')
+ 
+export const WishContext = createContext();
+
+export const WishProvider = ({ children }) => {
+  const [wish, setWish] = useState([]);
+
+  useEffect(() => {
+    axios.get('https://e-commerce-backend.harshporwal1.repl.co/wishlist')
+      .then(response => setWish(response.data))
+    // axios.post(`https://e-commerce-backend.harshporwal1.repl.co/wishlist/6086c1a338588900a6e3d68d`)
+    //   .then(response => console.log(response.data))
+  })
+
+  const addToWishList = (product) => {
+    wish.map(item => {
+      if (item.id === product.id) {
+        axios.delete(`https://e-commerce-backend.harshporwal1.repl.co/wishlist/${product.id}`)
```

Comment:
network calls are done in wrong way. Please revise concepts and also go through some tutorials of how to integrate frontend with backend. You are sending infinite request to server.

Reply:
fixed